flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStOptions.java

```diff
@@ -157,17 +157,27 @@ public class ForStOptions {
                                             + "of rocksdb timer service, but consumes more heap memory at the same time.",
                                     TIMER_SERVICE_FACTORY.key(), ForStDB.name()));
 
+    public static final ConfigOption<Boolean> EXECUTOR_WRITE_IO_INLINE =
```

Comment:
Add this option to `ForStConfigurableOptions#CANDIDATE_CONFIGS` to make it configurable?